### ISSUE TITLE
Unhandled exception. System.MissingMethodException: Method not found: 'System.String 

### ISSUE THREAD

**Issue by SanseLKagenhart** [CmlLib/CmlLib.Core.Installer.Forge#6]
```
using CmlLib.Core.Installer.Forge;
using CmlLib.Core;
using CmlLib.Core.Auth;
using CmlLib.Core.Installers;
using CmlLib.Core.ProcessBuilder;

var path = new MinecraftPath("./la"); // use default directory
var launcher = new MinecraftLauncher(path);

// show launch progress to console
var fileProgress = new SyncProgress<InstallerProgressChangedEventArgs>(e =>
    Console.WriteLine($"[{e.EventType}][{e.ProgressedTasks}/{e.TotalTasks}] {e.Name}"));
var byteProgress = new SyncProgress<ByteProgress>(e =>
    Console.WriteLine(e.ToRatio() * 100 + "%"));
var installerOutput = new SyncProgress<string>(e =>
    Console.WriteLine(e));

//Initialize variables with the Minecraft version and the Forge version
var mcVersion = "1.20.1";
var forgeVersion = "47.2.32";

//Initialize MForge
var forge = new ForgeInstaller(launcher);

var version_name = await forge.Install(mcVersion, forgeVersion, new ForgeInstallOptions
{
    FileProgress = fileProgress,
    ByteProgress = byteProgress,
    InstallerOutput = installerOutput,
});
//var version_name = await forge.Install(mcVersion); // install the recommended forge version for mcVersion
//OR var version_name = forge.Install(mcVersion, forgeVersion).GetAwaiter().GetResult();

//Start Minecraft
var launchOption = new MLaunchOption
{
    MaximumRamMb = 1024,
    Session = MSession.CreateOfflineSession("TaiogStudio"),
};

var process = await launcher.CreateProcessAsync(version_name, launchOption);

// print game logs
var processUtil = new ProcessWrapper(process);
processUtil.OutputReceived += (s, e) => Console.WriteLine(e);
processUtil.StartWithEvents();
await processUtil.WaitForExitTaskAsync();

Error:
Unhandled exception. System.MissingMethodException: Method not found: 'System.String CmlLib.Core.PackageName.GetPath(System.String, System.String)'.
   at CmlLib.Core.Installer.Forge.ForgeMapper.ToFullPath(String str, String prepath)
   at CmlLib.Core.Installer.Forge.ForgeInstallProcessor.MapProcessorData(JsonElement data, String minecraftJar, String libraryPath, String installDir)
   at CmlLib.Core.Installer.Forge.ForgeInstallProcessor.MapAndStartProcessors(String installerDir, String vanillaJarPath, String libraryPath, JsonElement installProfile, IProgress`1 progress, IProgress`1 processorOutput)
   at CmlLib.Core.Installer.Forge.Installers.FNewest.Install(MinecraftPath path, IGameInstaller installer, ForgeInstallOptions options)
   at CmlLib.Core.Installer.Forge.ForgeInstaller.Install(ForgeVersion forgeVersion, ForgeInstallOptions options)
   at CmlLib.Core.Installer.Forge.ForgeInstaller.Install(String mcVersion, String forgeVersion, ForgeInstallOptions options)
   at Program.<Main>$(String[] args) in C:\Users\Admin\source\repos\ConsoleApp1\ConsoleApp1\Program.cs:line 25
   at Program.<Main>(String[] args)```

**Comment by VoidTimeWithNull** [comment]
same problem and same copied code from the repository

**Comment by AlphaBs** [comment]
will be fixed on next version. for now, installing CmlLib.Core **4.0.0** will work

**Comment by AlphaBs** [comment]
fixed on 1.0.1
